FEATURES.md

```diff
@@ -339,38 +339,37 @@ A Map is desugared to a Map data type containing two constructors `Map/Leaf` and
 
 ```py
 # When you write this
-def empty_map:
+def empty_map() -> Map(T):
   return {}
 
-def init_map:
-  return { 1: "one", 2: "two", `blue`: 0x0000FF }
+def init_map() -> Map(String):
+  return { 1: "one", 2: "two"}
```

Comment:
Keep the `\`blue\`` key, it's meant to show the symbol syntax sugar (numeric value for small strings), just change the value to also be a string.